src/main/java/io/github/thebusybiscuit/slimefun4/core/multiblocks/MultiBlockMachine.java

```diff
@@ -165,6 +168,30 @@ public void load() {
         }
     }
 
+    /**
+     * This method handles an output {@link ItemStack} from the {@link MultiBlockMachine} which has a crafting delay
+     *
+     * @param outputItem
+     *          A crafted {@link ItemStack} from {@link MultiBlockMachine}
+     * @param container
+     *          Our main {@link Container} from {@link MultiBlockMachine}
+     *
+     */
+    @ParametersAreNonnullByDefault
+    protected void handleCraftedItem(ItemStack outputItem, Container container) {
+        Inventory containerInv = container.getInventory();
+        Inventory outputInv = findOutputInventory(outputItem, container.getBlock(), containerInv);
```

Comment:
`Container#getBlock()` is a relatively heavy operation as it does not cache the `Block` internally, whereas we already gotten the `Block` object before-hand, making this completely avoidable.

In other words: This `Block` object should probably be passed via a parameter here to reduce the performance impact of this method.